lib/types/concept.ts

```diff
@@ -0,0 +1,62 @@
+import { Link } from './link'
+import { LocaleCode } from './locale'
+
+type ISODateString = string
+
+export type ConceptSys = {
+  id: string
+  type: 'TaxonomyConcept'
+  createdAt: ISODateString
+  updatedAt: ISODateString
+  version: number
+}
+
+export interface Concept<Locales extends LocaleCode> {
+  sys: ConceptSys
+  uri?: string
+  prefLabel: {
+    [locale in Locales]: string
+  }
+  altLabels?: {
+    [locale in Locales]: string[]
+  }
+  hiddenLabels?: {
+    [locale in Locales]: string[]
+  }
+  note?: {
```

Comment:
```suggestion
  note?: {
```

A small note that all notes can be `null` as well